easybuild/easyconfigs/n/NCL/NCL-6.4.0-intel-2018a.eb

```diff
@@ -0,0 +1,47 @@
+name = 'NCL'
+version = '6.4.0'
+
+homepage = 'http://www.ncl.ucar.edu'
+description = "NCL is an interpreted language designed specifically for scientific data analysis and visualization."
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'cstd': 'c99', 'openmp': True, 'pic': True}
+
+# download via https://www.earthsystemgrid.org/dataset/ncl.html
+sources = ['%(namelower)s_ncarg-%(version)s.tar.gz']
+patches = ['NCL-%(version)s_fix-types.patch']
+checksums = [
+    'ebc9ec9538044cc4887c0c46adac8522242528d40c974e0b88b5b5d756f22a17',  # ncl_ncarg-6.4.0.tar.gz
+    'f6dfaf95e5de9045745e122cb44f9c035f81fab92f5892991ddfe93945891c8f',  # NCL-6.4.0_fix-types.patch
+]
+
+hdf5_ver = '1.10.1'
+
+dependencies = [
+    ('cURL', '7.55.1'),
+    ('JasPer', '1.900.1'),
+    ('g2lib', '1.4.0'),
+    ('g2clib', '1.6.0'),
+    ('HDF', '4.2.13'),
+    ('HDF5', hdf5_ver), 
+    ('netCDF', '4.6.0', '-HDF5-%s' % hdf5_ver),
+    ('netCDF-Fortran', '4.4.4', '-HDF5-%s' % hdf5_ver),
+    ('Szip', '2.1.1'),
+    ('freetype', '2.9'),
+    ('zlib', '1.2.11'),
+    ('GDAL', '2.2.3', '-Python-2.7.14-HDF5-%s' % hdf5_ver),
+    ('UDUNITS', '2.2.26'),
+    ('ESMF', '7.0.2'),
+    ('bzip2', '1.0.6'),
+    ('cairo', '1.14.12'),
+    ('libiconv', '1.15'),
+    ('GSL', '2.4'),
+]
+builddependencies = [('makedepend', '1.0.5')]
+
+sanity_check_paths = {
+    'files': ['bin/fontc', 'lib/libncarg.a'],
+    'dirs': ['include/ncarg', 'lib/ncarg/fontcaps'],
```

Comment:
this is handled in the `NCL` easyblock?

Reply:
Some of the targets are not created so without this update the easybuild fails.

Reply:
@SimonPinches That probably means the installation is partially failing for some reason...

If `bin/ncl` isn't there, something is very wrong, so please don't bypass the sanity check but fix the compilation problems.

If you need help, please provide a (debug) log for NCL.

Reply:
Thanks @boegel.  Here's the log if it helps figure out what's going wrong:

[easybuild-NCL-6.4.0-20180704.170005.uzchd.log](https://github.com/easybuilders/easybuild-easyconfigs/files/2163819/easybuild-NCL-6.4.0-20180704.170005.uzchd.log)

Reply:
This seems to be part of the problem:

```
NclNewHDF5.c(1499): error #165: too few arguments in function call
        dataset_id = H5Rdereference(attr_id, H5R_OBJECT, &rbuf[i]);
```

This probably means that `NCL` 6.4.0 isn't compatible with `HDF5` 1.10.x, it seems to require `HDF5` 1.8.x...